README.md

```diff
@@ -1,9 +1,16 @@
+![Build Status](https://travis-ci.org/dockstore/write_api_service.svg?branch=develop)
```

Comment:
This still requires more explanation. 
Imagine yourself as a developer with a passing knowledge of Dockstore that comes across this. 
Explain, what this tool is used for, who needs to use this tool, what this tool replaces, etc.